scripts/network-spec

```diff
@@ -7,5 +7,5 @@ NETWORK_NAME=DPoSChain
 NETWORK_ID=101
 OWNER=0x1092a1E3A3F2FB2024830Dd12064a4B33fF8EbAe
 INITIAL_VALIDATORS=0xbbcaa8d48289bb1ffcf9808d9aa4b1d215054c78,0x75df42383afe6bf5194aa8fa0e9b3d5f9e869441,0x522df396ae70a058bd69778408630fdb023389b2
-STAKING_EPOCH_DURATION=120960
-COLLECT_ROUND_LENGTH=200
+STAKING_EPOCH_DURATION=50
```

Comment:
Don't we need three random numbers if we have three validators? I.e. should `STAKING_EPOCH_DURATION` always be at least `3 * COLLECT_ROUND_LENGTH`?

Reply:
Actually, in production mode we need 20 random numbers (max number of validators): https://github.com/poanetwork/pos-contracts/blob/924b61525eec1ee38afa4a6ec8d48850343dbadd/contracts/abstracts/ValidatorSetBase.sol#L623 - but this condition works only for the case when the number of candidates is greater than `MAX_VALIDATORS` (greater than 20): https://github.com/poanetwork/pos-contracts/blob/924b61525eec1ee38afa4a6ec8d48850343dbadd/contracts/abstracts/ValidatorSetBase.sol#L616

For the current testing mode, it is enough to have less than 20 candidates. Maybe I'll make `MAX_VALIDATORS` constant configurable later to be able to decrease its value for testing purposes (to test random choosing).